test/users/npadmana/blas/c2chapel.py

```diff
@@ -0,0 +1,99 @@
+""" A C to Chapel parser to help automate writing bindings.
```

Comment:
Is there a reason this is in the test directory? Maybe `util/devel` would be a better home for this script.